### PR TITLE
Choreo Connect related changes regarding Choreo-Organization-Purge Feature

### PR DESCRIPTION
**Pull request by rashm1n**
### Purpose
$Subject

### Related Issue
* https://github.com/wso2-enterprise/choreo/issues/7058

### CODE REVIEW

File: adapter/pkg/messaging/azure_connection.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	logger "github.com/wso2/product-microgateway/adapter/internal/loggers"
 )
 
-var bindingKeys = []string{tokenRevocation, notification, stepQuotaThreshold, stepQuotaReset}
+var bindingKeys = []string{tokenRevocation, notification, stepQuotaThreshold, stepQuotaReset, organizationPurge}
```
Comment (mpmunasinghe) [inline adapter/pkg/messaging/azure_connection.go:32]: @Rajith90 AFAIR you had a concern with this array based approach for connection keys?

Comment (Rajith90) [reply adapter/pkg/messaging/azure_connection.go:32]: Yes, we decided to refactor this when @erandi refactors the rate limiting related topics

**Review by mushthaq33 — APPROVED** [review]

File: adapter/pkg/messaging/azure_connection.go (hunk 1 above)
Comment (Rajith90) [reply adapter/pkg/messaging/azure_connection.go:32]: @erandiganepola could you please follow up on this

**Review by Rajith90 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20210913.18

**Comment by choreo-cicd** [comment]
[failed] stage-deployment : 20210916.1

**Comment by choreo-cicd** [comment]
[failed] prod-deployment : 20210917.1

**Pull request merged**